pkg/omf/omf.fish

```diff
@@ -119,13 +123,18 @@ function omf -d "Oh My Fish"
 
     case "t" "theme"
       if test (count $argv) -eq 1
-        set -l ostype (uname)
         set -l theme (cat $OMF_CONFIG/theme)
-        set -l regex "[[:<:]]($theme)[[:>:]]"
-        test "$ostype" != "Darwin"; and set regex "\b($theme)\b"
-
-        omf.packages.list --database --theme | column | sed -E "s/$regex/"(omf::em)"\1"(omf::off)"/"
-        omf::off
+        set -l highlight_current "(^|\s)($theme)(\s|\$)"
```

Comment:
It should be a word boundary, only that `\b` also matches hyphens (-), so that choosing _agnoster_ highlighted both **agnoster** and **agnoster**-mercurial.
So it's  -- either the beginning of the line - or - a whitespace character -+- the name of the theme -+- a whitespace character - or - the end of the line --.
And of course `$` marks the end of the line but has to be escaped in the shell so that Fish doesn't expect a variable name.